packages/dev-server/test-plugins/reviews/entities/product-review-translation.entity.ts

```
@@ -1,11 +1,16 @@
+
+export class CustomReviewFieldsTranslation {}
 
 @Entity()
-export class ProductReviewTranslation extends VendureEntity implements Translation<ProductReview> {
+export class ProductReviewTranslation
+    extends VendureEntity
+    implements Translation<ProductReview>, HasCustomFields
+{
     constructor(input?: DeepPartial<Translation<ProductReviewTranslation>>) {
         super(input);
     }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)